styles.css

```diff
@@ -400,6 +400,88 @@ header {
     position: absolute;
     flex-direction: column;
   }
+
+  .project-description {
+    font-size: 17px;
+  }
+
+  .works {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 24px;
+  }
+
+  .works-title-section {
+    display: flex;
+    align-items: center;
+    padding-top: 17px;
+    padding-bottom: 42px;
+    grid-column: 1 / 4;
+  }
+
+  .works-title-section div {
+    width: calc(100% - 339px);
+  }
+
+  .works-title {
+    width: 315px;
+    margin-right: 24px;
+    height: 52px;
+    color: #091e42;
+    font-weight: 400;
+  }
+
+  .featured-project {
+    display: flex;
+    grid-column: 1 / 4;
+  }
+
+  .f-project-img {
+    width: 648px;
+    height: 448px;
+    margin-right: 24px;
+  }
+
+  .f-project-body {
+    width: calc(100% - 648px);
+  }
+
+  .f-project-body > .project-title {
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 0;
+    color: #091e42;
+  }
+
```

Comment:
- Good job aligning your project cards 👍 There are a few issues to address here. 
- [ ] Each project should have their description on top of their image with a dark opacity and with a white color text as shown in the second project card. 
- [ ] When the project is hovered on with the mouse the description and dark opacity disappear and a `See Project` button appears to the bottom of the image. Kindly implement this feature so as to follow the template design. Hence all projects should look like the first project card when you hover over it.
- Your design:
![Annotation 2022-03-23 200025](https://user-images.githubusercontent.com/85113567/159778540-d9fb4eb4-93d7-4bc2-b86e-a1fd21e88bcd.jpg)
- Figma design:
![Annotation 2022-03-23 200039](https://user-images.githubusercontent.com/85113567/159778583-a03c7d24-f6c6-4126-9eb1-4eab17b63a90.jpg)